From 6dd831eb5909d2cc60ec434be45186449ded0c2d Mon Sep 17 00:00:00 2001
From: Basil Chew <basil.chew@intel.com>
Date: Thu, 15 Sep 2022 12:51:08 +0800
Subject: [PATCH] Update host setup to QEMU 7.1

Change:
- updated setup_host.sh to install QEMU 7.1
- ensured submodules are updated before patching

Change-Id: I0ec881407b9f8ff1ea99858c8887caacf057db94

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index 372b9f7..a812987 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -9,7 +9,7 @@ set -eE
 
 #---------      Global variable     -------------------
 reboot_required=0
-QEMU_REL="qemu-7.0.0"
+QEMU_REL="qemu-7.1.0"
 
 #Directory to keep versions of qemu which can be reused instead of downloading again
 QEMU_CACHE_DIR="$HOME/.cache/civ/qemu"
@@ -114,6 +114,7 @@ function ubu_build_ovmf_gvt(){
     git clone https://github.com/tianocore/edk2.git
     cd $CIV_WORK_DIR/edk2
     git checkout -b stable202111 edk2-stable202111
+    git submodule update --init
 
     patch -p1 < $CIV_WORK_DIR/patches/ovmf/0001-OvmfPkg-add-IgdAssignmentDxe.patch
     if [ -d $CIV_GOP_DIR ]; then
@@ -129,8 +130,6 @@ function ubu_build_ovmf_gvt(){
         done
     fi
 
-    git submodule update --init
-
     source ./edksetup.sh
     make -C BaseTools/
     build -b DEBUG -t GCC5 -a X64 -p OvmfPkg/OvmfPkgX64.dsc -D NETWORK_IP4_ENABLE -D NETWORK_ENABLE  -D SECURE_BOOT_ENABLE -D TPM_ENABLE
-- 
2.38.0

